### PR TITLE
feat: asset cards visual updates

### PR DESCRIPTION
**Pull request by dzbo**
### Description

This is base feature branch that will contain all visual changes around assets. Smaller, per issue, PR's will be merged into this. This will make much better review, test and develop experience.

### CODE REVIEW

File: translations/en_US.json [hunk 1]
```diff
@@ -12,11 +12,13 @@
   "token_details_contract_address": "Contract Address",
   "landing_hero_description_no_extension": "Looks like you don’t have the <span class=\"paragraph-inter-16-semi-bold\">Universal Profile browser extension</span> installed. You’ll need it before you can log in to our wallet service.",
   "error_no_accounts": "Account not found.",
+  "asset_all_creators_unverified": "Un-verified creators",
   "modal_switch_application_network_title": "Change application network",
   "send_pending_title": "Sending {amount} {symbol}",
   "buy_lyx_title": "Use the below services to <strong>buy LYX</strong> for your Universal Profile",
   "assets_empty_state_title": "No created assets",
   "web3_connect_no_extension": "You don't have the Universal Profile browser extension installed.",
+  "asset_all_creators_partial": "Not all creators verified",
```
Comment (Hugoo) [inline translations/en_US.json:21]: maybe

```suggestion
  "asset_all_creators_partial": "Not all creators are verified",
```

File: utils/paddedToAddress.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/**
+ * Remove padding from address
+ *
+ * @param address
+ * @returns
+ */
+export const paddedToAddress = (address: string) => {
+  if (address.slice(0, 2) === '0x') {
```
Comment (Hugoo) [inline utils/paddedToAddress.ts:8]: maybe

```suggestion
  if (address.startsWith('0x')) {
```

not a big deal but i think it reads easier

File: utils/paddedToAddress.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    address = address.slice(2)
+  }
+
+  while (address.slice(0, 2) === '00') {
+    address = address.slice(2)
+  }
```
Comment (Hugoo) [inline utils/paddedToAddress.ts:14]: For the case of:

`0x000000000000000000000000002419a44c90ff9dd06ce378115e0bf32442d4a5`

You can do a simple:

replace `0x000000000000000000000000` by `0x`

The current script will break if the address starts with 00 which can happen. (it didnt and i was wrong here: https://github.com/lukso-network/wallet.universalprofile.cloud/pull/251#discussion_r1477030036 )

In that case it is always 32bytes to 20bytes so you need to remove 12 bytes so 24 times 0

`0x000000000000000000000000065682c3c4300e75e1a67837105e2ae93e1ff575`

->

`0x065682c3c4300e75e1a67837105e2ae93e1ff575`

https://explorer.execution.mainnet.lukso.network/address/0x065682c3c4300e75e1a67837105e2ae93e1ff575

For some reasons the burnt pix starts with 0.....

**Review by Hugoo — APPROVED** [review]
i approve but the token address convert has an issue

File: utils/paddedToAddress.ts (hunk 3 above)
Comment (dzbo) [reply utils/paddedToAddress.ts]: I've added additional `isAddress` check and tests so it will work with addresses starting with `0` too.

Comment (Hugoo) [reply utils/paddedToAddress.ts]: Why not just a searchAndReplace? but ok this will work

**Pull request merged**